app/backend/src/couchers/migrations/versions/c035cdcabd40_add_new_events_tables.py

```diff
@@ -75,7 +75,7 @@ def upgrade():
         op.f("ix_cluster_event_associations_event_id"), "cluster_event_associations", ["event_id"], unique=False
     )
     op.create_table(
-        "event_occurences",
+        "event_occurrences",
```

Comment:
Don't touch old migrations! (your own words :) )

Reply:
This is a new migration in this PR so it's fine. But yes, shouldn't touch migrations already on `develop` (or a bit more leniently, those that have been applied in prod)

Reply:
My mistake looking at the last commit only 😊